new.go

```diff
@@ -291,3 +291,20 @@ func prepareNewWindowsImageIfNeeded(image *CNBImageCore) error {
 	}
 	return nil
 }
+
+func NewCNBIndex(repoName string, v1Index v1.ImageIndex, ops IndexOptions) (*CNBIndex, error) {
```

Comment:
Why are we providing `v1Index` as a separate argument when it is contained in `ops`?

Reply:
I think at some point I wanted to make mandatory that if you want to create a CNBIndex you need a v1.ImageIndex during creation, but we do [this](https://github.com/buildpacks/imgutil/pull/191/files/e9716dbb3f7abae4ff8986ac44cb0a147c4cf815#r1575089124) then we don't need it anymore

Reply:
Resolved by https://github.com/husni-faiz/imgutil/pull/12